.gitignore

```diff
@@ -6,3 +6,4 @@
 /dist/
 /django_scheduler.egg-info/
 /docs/_build/
+.idea/
```

Comment:
Can you remove this line? It looks unrelated to the PR of adding object level permissions.